poem.txt

```diff
@@ -8,6 +8,6 @@ beige
 ivory
 khaki
 these are the colours of the rainbow.
-
- (I spelt colours correctly the first time so this message will be my edit.)
-
+Black Black Blak
+Is blak a color or is it a shad?
```

Comment:
Black misspelt (change Blak to Black)
colour misspelt (change color to colour)
shade misspelt (change shad to shade)